command.go

```diff
@@ -9,15 +9,20 @@ import (
 )
 
 type Command struct {
-	Name     string
-	Interval time.Duration
-	Timeout  time.Duration
-	Exec     string
-	Params   []string
-	Enabled  bool
-	lastRun  time.Time
+	Name           string
+	Interval       time.Duration
+	Timeout        time.Duration
+	Exec           string
+	Params         []string
+	Enabled        bool
+	SecondsOverdue time.Duration
+	lastRun        time.Time
 }
 
+const (
```

Comment:
I think we should rather make a function to express this concept:

```go
func (c* Command) isDailyCommand() bool { return c.Interval == 24 * time.Hour }
```

Since we're adding some significant voodoo for the magic interval "24 hours", we should make that very clear. When refactoring, it's much easier to miss a Interval == 24 vs a function with an explicit name.